relm-derive/src/gen/parser.rs

```diff
@@ -400,25 +404,35 @@ impl Parse for Attribute {
 
 struct Attributes {
     name_values: HashMap<String, Option<LitStr>>,
+    style_classes: HashSet<String>,
 }
 
 impl Parse for Attributes {
     fn parse(input: ParseStream) -> Result<Self> {
         let mut name_values = HashMap::new();
+        let mut style_classes = HashSet::new();
         loop {
             let lookahead = input.lookahead1();
 
             if lookahead.peek(Token![#]) {
                 let attribute: Attribute = input.parse()?;
-                name_values.extend(attribute.name_values);
+                match attribute.name_values.get("style_class") {
+                    Some(Some(style_class)) => {
+                        style_classes.insert(style_class.value());
+                        ()
```

Comment:
This line should not be necessary.

Reply:
you mean the `()`? the problem is that the `match` wants every branch to return the same type, and the other branch returns `()` while the insert returns `bool`. I quite believe there's a nicer way to fix that, but I don't know it.

Reply:
Yeah, I've seen this issue, but usually, having the curly braces and the semi-colon should be enough. Are you sure it's not working when you remove the `()`?

Reply:
you are right, i could just remove it, pushed now in a followup commit.